test/test_jewish_calendar.py

```diff
@@ -677,6 +677,202 @@ def test_day_of_omer_outside_of_omer(self):
         calendar = JewishCalendar(test.test_helper.standard_monday_chaseirim(), 7, 1)
         self.assertIsNone(calendar.day_of_omer())
 
+    def test_significant_shabbes_for_standard_monday_chaseirim(self):
```

Comment:
looks great, please fix the spelling for 'shabbos' in the test method names to match the rest of the code